content/open-source/sigstore/policy-controller/how-to-install-policy-controller.md

```diff
@@ -0,0 +1,165 @@
+* **kubectl** — to work with your cluster. Install `kubectl` for your operating system by following the official [Kubernetes kubectl documentation](https://kubernetes.io/docs/tasks/tools/#kubectl).
+* The [Helm](https://helm.sh) command line tool to install the Policy Controller.
+
+If you are using the terminal that is embedded on this page, then all the prerequsites are installed for you.
+
+Once you have everything in place you can continue to the next step and install the Policy Controller.
+
+## Step 1 — Creating the `cosign-system` Kubernetes Namespace
+
+The first step that you need to complete is to create a Kubernetes `namespace` for the Policy Controller to run in. Call it `cosign-system` and run the following command to create it:
+
+```bash
+kubectl create namespace cosign-system
+```
+
+Now you can move on to the next step, which is installing the Policy Controller into the namespace you just created.
+
+## Step 2 — Installing the Policy Controller
+
+In this step we'll use the `helm` command line tool to install the Policy Controller.
+
+First, add the [Sigstore Helm Repository](https://sigstore.github.io/helm-charts) to your system with the following command:
+
+```bash
+helm repo add sigstore https://sigstore.github.io/helm-charts
+```
+
+You should receive output like this:
+
+```
+"sigstore" has been added to your repositories
+```
+
+Next, update your local Helm repository information using the `helm repo update` comand:
+
+```bash
+helm repo update
+```
+
+You'll receive output like the following:
+
+```
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "sigstore" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+
+Now install the Policy Controller into the `cosign-system` namespace that you created in the first step of this guide:
+
+```bash
+helm install policy-controller -n cosign-system sigstore/policy-controller --devel
+```
+
+The `--devel` flag will include any alpha, beta, or release candidate versions of a chart. You can specify a particular version with the `--version` flag if you prefer.
+
+It may take a few minutes for your cluster to deploy all of the manifests needed for the Policy Controller. Check the status of your cluster using the `kubectl wait` command like this:
+
+```bash
+kubectl -n cosign-system wait --for=condition=Available deployment/policy-controller-webhook && \
+kubectl -n cosign-system wait --for=condition=Available deployment/policy-controller-policy-webhook
+```
+
+Once the Policy Controller deployments are done you will receive output like the following:
+
+```
+deployment.apps/policy-controller-webhook condition met
+deployment.apps/policy-controller-policy-webhook condition met
+```
+
+A full list of the resources that the Policy Controller deploys into your cluster is available at the end of this guide in [Appendix 1 — Resource Types](#appendix-1--resource-types).
```

Comment:
I would call it just Appendix, since there is only one. Maybe I would style it as `Apendix: Resource Types` but maybe the em dash keeps it more consistent